justfile

```diff
@@ -5,8 +5,23 @@ default:
     @just --list
 
 # Update the OpenApi yml files by generating these from the tAPIr specs
-alias dog := docs-openapi-generate
 docs-openapi-generate:
-    mkdir -p {{openapiDir}}
-    rm {{openapiDir}}/*.yml >> /dev/null 2>&1 || true
-    sbt "webapi/runMain org.knora.webapi.slice.common.api.DocsGenerator {{openapiDir}}"
+    mkdir -p {{ openapiDir }}
+    rm {{ openapiDir }}/*.yml >> /dev/null 2>&1 || true
+    sbt "webapi/runMain org.knora.webapi.slice.common.api.DocsGenerator {{ openapiDir }}"
+
+alias dog := docs-openapi-generate
+
+# Start stack from latest images on dockerhub (no build required)
+start-stack-latest:
```

Comment:
I would like to propose to rename all stack related recipes so that they all start with `stack-*`.

So when you call `just` you have them all nicely ordered underneath each other.

Like so:
```
stack-start:
  …
stack-stop:
  …
```

Maybe also add a destroy recipe?
```
# Stops and removes all stack related container volumes and network
stack-destroy: 
  echo "Destroying Stack"
  echo "\nStopping and removing containers"
  docker stop $(docker ps -aq)
  docker rm $(docker ps -aq)
  echo "\nPruning Network"
  docker network rm knora-net 
  echo "\nPruning Volume"
  docker volume rm dsp-api_db-home -f
  docker volume rm dsp-api_db-import -f
  echo "Stack destroyed"
```


``` 
 stack-init:
    …
```

Reply:
see [Add more recipes for starting, stopping, destroying, initialization o…](https://github.com/dasch-swiss/dsp-api/pull/2985/commits/4d51f4641f30d8f05abbdbbf7c34643e1aaf5e66)